test/e2e/user_workloads_test.go

```diff
@@ -71,22 +73,36 @@ func (s *userWorkloadsTestSuite) TestIdlerAndPriorityClass() {
 		err := memberAwait.WaitUntilPodsDeleted(p.Namespace, wait.WithPodName(p.Name))
 		require.NoError(s.T(), err)
 	}
+	// check notification was created and Idler condition was updated
+	_, err = memberAwait.WaitForIdler("test-idler-dev", wait.IdlerConditions(Running(), IdledNotificationCreated()))
+	require.NoError(s.T(), err)
```

Comment:
The condition is an implementation detail here. What you need is to check that the notification is created (and this is what you already do below). How it's done (via conditions or something else) is an implementation detail we are not interested in the e2e tests.

```suggestion
	// check notification was created
```

Reply:
done in [43d63bd](https://github.com/codeready-toolchain/toolchain-e2e/pull/560/commits/43d63bdc215c35b00065d694a62354fbce6aa696)